db/migrate/20180611075005_add_collection_ref_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddCollectionRefToUsers < ActiveRecord::Migration[5.1]
```

Comment:
Probably not worth fixing, but this migration name is misleading. It is really adding the user reference to the collections table.